lib/src/modules/conditional.dart

```diff
@@ -20,7 +20,7 @@ import 'package:rohd/src/utilities/uniquifier.dart';
 /// Represents a block of logic, similar to `always` blocks in SystemVerilog.
 abstract class _Always extends Module with CustomSystemVerilog {
   /// A [List] of the [Conditional]s to execute.
-  final List<Conditional> conditionals;
+  late List<Conditional> conditionals;
```

Comment:
Can we change this to use an `UnmodifiableListView` instead?

```dart
List<Conditional> get conditionals => UnmodifiableListView<Conditional>(_conditionals);
late List<Conditional> _conditionals;
```

Otherwise it opens the door to people re-assigning the list after construction, which is weird and might cause some unpredictable behavior.